include/roboteam_world/kalman/WorldFilter.h

```diff
@@ -0,0 +1,35 @@
+#ifndef ROBOTEAM_WORLD_KALMANFILTER_H
+#define ROBOTEAM_WORLD_KALMANFILTER_H
+
+#include <kalman/RobotFilter.h>
+#include <kalman/BallFilter.h>
+#include "roboteam_utils/Position.h"
+#include "roboteam_proto/WorldRobot.pb.h"
+#include "roboteam_proto/World.pb.h"
+
+namespace world {
+
+//This class is a manager for the different Kalman object classes
```

Comment:
OK so what does it manage exactly? This does not make things easier for me to understand

Reply:
Well the different objects being tracked (e.g. it determines when to create filters, when to delete them and which filter is picked to send information to roboteam_AI)